### PR TITLE
Create settings file and show the proximity radius by default  issues #1, #2, #3

### PR DESCRIPTION
**Pull request by imwongela**
- Creates a settings file
  - Disable proximity if GPS is not enabled on the phone
- Show the proximity radius by default.

### CODE REVIEW

**Comment by dalekunce** [comment]
All of this looks reasonable. @hallahan would you mind reviewing and then merging in.

File: app/src/main/java/org/redcross/openmapkit/MapActivity.java [hunk 1]
```diff
@@ -135,27 +159,40 @@ protected void positionMap() {
...
             LatLng c = new LatLng(lat, lng);
             mapView.setCenter(c);
             mapView.setZoom(z);
+            mapView.setUserLocationEnabled(true);
         }
     }
-    
+
+    protected void addGeojsonOverlay() {
+        mapView.loadFromGeoJSONURL("http://api.mspray.onalabs.org/buffers.json?target_area=408_257");
```
Comment (hallahan) [inline app/src/main/java/org/redcross/openmapkit/MapActivity.java]: What is the hard coded JSON here? Should we put this in a settings file maybe?

**Comment by hallahan** [comment]
This looks really cool!

File: app/src/main/java/org/redcross/openmapkit/MapActivity.java (hunk 1 above)
Comment (mberg) [inline app/src/main/java/org/redcross/openmapkit/MapActivity.java]: Apologies this is something that shouldn't be here.  Was a test of mapbox
sdk geojson rendering.  Will ask Isaac to revert and do another pull
request.

On Sun, Jul 19, 2015 at 8:39 PM, Nicholas Hallahan <notifications@github.com

Comment (mberg) [inline app/src/main/java/org/redcross/openmapkit/MapActivity.java]: Or Nick feel free to remove that and merge it in.  Thanks!

Comment (hallahan) [inline app/src/main/java/org/redcross/openmapkit/MapActivity.java]: I'll just remove it for you then.

**Comment by hallahan** [comment]
I'm thinking we need to move the LocationXMLParser out of the overlays module of the MapboxAndroidSDK. I think this is specific to OpenMapKit, not a mapping library thang.

I'm going to merge this request in and then made a few tweaks.

File: MapboxAndroidSDK/src/main/java/com/mapbox/mapboxsdk/views/MapView.java [hunk 2]
```diff
@@ -1792,16 +1792,10 @@ private UserLocationOverlay getOrCreateLocationOverlay() {
     }
 
     /**
-     * Show or hide the user location overlay
+     * Show the user location overlay
      */
     public MapView setUserLocationEnabled(final boolean value) {
```
Comment (hallahan) [inline MapboxAndroidSDK/src/main/java/com/mapbox/mapboxsdk/views/MapView.java]: I see why this was done, but I think it's important to be able to toggle user location when _not_ in proximity mode. I'm just going to merge this in for now and make an issue.

**Pull request merged**